cmd/rainbow/main.go

```diff
@@ -91,69 +92,80 @@ func run(args []string) {
 		&cli.StringSliceFlag{
 			Name:    "next-crawler",
 			Usage:   "forward POST requestCrawl to this url, should be machine root url and not xrpc/requestCrawl, comma separated list",
-			EnvVars: []string{"RELAY_NEXT_CRAWLER"},
+			EnvVars: []string{"RAINBOW_NEXT_CRAWLER", "RELAY_NEXT_CRAWLER"},
+		},
+		&cli.StringFlag{
+			Name:    "env",
+			Usage:   "operating environment (eg, 'prod', 'test')",
+			Value:   "dev",
+			EnvVars: []string{"ENVIRONMENT"},
+		},
+		&cli.BoolFlag{
+			Name:  "enable-otel-otlp",
+			Usage: "enables OTEL OTLP exporter endpoint",
+		},
+		&cli.StringFlag{
+			Name:    "otel-otlp-endpoint",
+			Usage:   "OTEL traces export endpoint",
+			Value:   "http://localhost:4318",
+			EnvVars: []string{"OTEL_EXPORTER_OTLP_ENDPOINT"},
```

Comment:
Should this just have no default value and then disable itself when not supplied? I'm not sure having a default here is useful to us since we generally don't run otel collectors on localhost.

Reply:
I added the separate `enable-otel-otlp` flag to enable/disable.

Previously the whole feature was gated by whether this config was enabled at all. The config was buried in code, and not obvious what kind of string to configure. This way running `./rainbow serve --help` displays all the env vars, and comes with an example value, which makes it more accessible. (at least is the theory)